pkg/controller/plan/kubevirt.go

```diff
@@ -950,6 +931,116 @@ func (r *KubeVirt) DeleteHookJobs(vm *plan.VMStatus) (err error) {
 	return
 }
 
+// Delete any populator resources that belong to a VM migration.
```

Comment:
I agree it's simplify the cleanup. But it will require additional changes in the populator controller and the CRDs. I don't know if we wish to have them (thinking of CDI usage). WDYT?